script/merge_dependencies.js

```diff
@@ -25,8 +25,13 @@ fs.writeFileSync(
         ...core.devDependencies,
       },
       prettier: { ...app.prettier, ...core.prettier },
+      packageManager: core.packageManager,
     },
     null,
     2
   )
 );
+
+const yarnRcCore = fs.readFileSync("./homeassistant-frontend/.yarnrc.yml", "utf8");
+const yarnRcApp = yarnRcCore.replace(/\.yarn\//g, "homeassistant-frontend/.yarn/");
```

Comment:
Regex is a better way to replace the yarn path for sure. We should use this above too. Can we set these constants: 
```
const yarn_regex = /\.yarn\//g
const yarn_path = "homeassistant-frontend/.yarn/" 
```
And also use these to replace this loop:
```
for (let k in core.resolutions) {
  core.resolutions[k] = core.resolutions[k].replace(
    "./.yarn/",
    "./homeassistant-frontend/.yarn/"
  );
}
```